### ISSUE TITLE
Annotation is added for all hours of the day

### ISSUE THREAD

**Issue by macobo** [PostHog/posthog#6325]
## Bug description

![image](https://user-images.githubusercontent.com/148820/136525282-be1d80df-8792-4470-8a1f-e4cc28a3db94.png)

## Expected behavior

Only add annotation to a specific hour/only once

## How to reproduce

1. Go to insights
2. Set interval to hourly
3. Add annotation
4. See annotation pop up under every hour

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

**Comment by neilkakkar** [comment]
Duplicate of https://github.com/PostHog/posthog/issues/6084
